docs/4.0/i18n/zh-Hans/examples/gitea/install-gitea.md

```diff
@@ -0,0 +1,120 @@
+# 快速安装 Gitea
+
+Gitea 是一个自己托管的 Git 服务程序。他和 GitHub, Bitbucket or Gitlab 等比较类似。Gitea 的首要目标是创建一个极易安装，运行非常快速，安装和使用体验良好的自建 Git 服务。
```

Comment:
![image](https://github.com/labring/sealos/assets/72536832/1f1dec18-8f4e-4ead-b41d-f49e52b45b37)

I think this is better.

Reply:
yeah,I think so